Dockerfile

```diff
@@ -58,6 +58,9 @@ COPY --chown=rails:rails --from=build /rails /rails
 # Entrypoint prepares the database.
 ENTRYPOINT ["/rails/bin/docker-entrypoint"]
 
+HEALTHCHECK --interval=30s --timeout=5s --start-period=10s --retries=3 \
```

Comment:
Rather than `curl` a full HTTP request can we do a TCP check to make sure the container is up?  Always hard to decide how far to decide "healthy" app ... roundtrip to the DB included?